collector/src/lib.rs

```diff
@@ -246,7 +255,8 @@ pub struct MasterCommit {
 /// Note that this does not contain try commits today, so it should not be used
 /// to validate hashes or expand them generally speaking. This may also change
 /// in the future.
-pub async fn master_commits() -> Result<Vec<MasterCommit>, Box<dyn std::error::Error + Sync + Send>> {
+pub async fn master_commits() -> Result<Vec<MasterCommit>, Box<dyn std::error::Error + Sync + Send>>
+{
```

Comment:
My editor runs cargo fmt on save, and it looks like this file wasn't fully formatted. Shall we add checking for formatting to the CI?